src/components/v5/common/ActionSidebar/partials/forms/PaymentBuilderForm/partials/FileUploadModal/types.ts

```diff
@@ -0,0 +1,12 @@
+import { type ModalProps } from '~v5/shared/Modal/types.ts';
+
+export interface FileUploadModalProps extends ModalProps {
+  onUpload: (file: CSVFileItem[]) => void;
+}
+
+export interface CSVFileItem {
+  recipient: string;
+  tokenContractAddress: string;
+  amount: string;
+  claimDelay: string;
+}
```

Comment:
I've left this type because it's needed because parsed file have this structure but I have used this type `ExpenditurePayoutFieldValue` in other places